plugins/GSdx/GS.cpp

```diff
@@ -967,7 +965,7 @@ EXPORT_C GSsetFrameSkip(int frameskip)
 
 EXPORT_C GSsetVsync(int enabled)
 {
-	s_vsync = !!enabled;
+	s_vsync = enabled;
```

Comment:
Parameter name should most likely be updated (Similar to the way you've done on the GSDevice* files)